docs/components/progress_indicators.md

```diff
@@ -0,0 +1,232 @@
+<!--docs:
+title: "Material progress indicators"
+layout: detail
+section: components
+excerpt: "Progress indicators express an unspecified wait time or display the length of a process."
+iconId:
+path: /catalog/material-progress-indicators/
+-->
+
+# Progress indicators
+
+[Progress indicators](https://material.io/components/progress-indicators) express an unspecified wait time or display the length of a process.
+
+![Progress indicator hero](assets/progress_indicators/hero-1.gif)
+
+## Contents
+
```

Comment:
This should be consistent with the contents of other articles in terms of heading 2/3/4

Reply:
Remove #contents link, and I think linear and circular should not be here, since they'd be ### and Anatomy and key properties would be ####

Reply:
Done